content/docs/rfcs/13/README.md

```diff
@@ -42,11 +42,11 @@ message Index {
 }
 
 message PagingInfo {
-  int64 pageSize = 1;
+  uint64 pageSize = 1;
   Index cursor = 2;
   enum Direction {
-    FORWARD = 0;
-    BACKWARD = 1;
+    FORWARD = 1;
```

Comment:
Maybe this was discussed and I missed it, but would FORWARD as 0 make more sense? IIRC that would be the default, so if you don't specify it you'd get FORWARD, which might be the more common use case?

Reply:
When I saw 'backward' I assumed the usecase is first page is latest messages and then go backward from there until you caught up.

Reply:
BACKWARD makes sense to me as the default, but there might have been an original design intent that I'm unaware of?

Reply:
Protobuf enum ordering should have default element first, though, regardless of value, to maintain compatibility between proto2 and proto3 compilers.

Reply:
Right, BACKWARDS makes more sense as default!